app/services/userService.js

```diff
@@ -0,0 +1,32 @@
+import uuid from 'uuid/v4'
+import db from '../../database/models'
+import encryptPassword from '../../lib/helpers/encrypt'
+
+class UserService {
+  static async createUser(req, res) {
+    {
```

Comment:
Just realized, we've got curly braces for function createUser... Do they have a purpose?